.github/workflows/test-push-workflows-service-image.yml

```
@@ -315,22 +259,22 @@
             "SHORT_SHA=${{ needs.build-and-push-image.outputs.sha_short }}"
 
   deploy-to-dev:
-    needs: build-and-push-ee-image
+    needs: [build-and-push-image,build-and-push-ee-image]
```

yamllint

[warning] too few spaces after comma

(commas)